stacktrace.js

```diff
@@ -157,6 +158,16 @@
             }
         },
 
+          /**
+         * Caches an instance of StackTraceGPS to ensure that source maps can use it's client side cache
+         *
+         * @param opts {Object}
+         */
+        createGlobalGPSInstance: function StackTrace$$(opts) {
```

Comment:
whoops ya lol, i'll fix these nits. the opts thing occurred to me, but from what i can tell the opts passed to fromError don't share properties with the opts for stacktrace gps. i def could have missed something there, but if so we could just have them be two separate sets of opts so the fromError opts would have no effect on stackTraceGPS. i realize that could be a breaking change , but for people using this new function they should know the opts they originally used are now cached so any opts passed to fromError have no effect on the GPS instance. unless you have a better thought there